packages/synapse-react-client/src/components/HasAccess/HasAccessV2.tsx

```diff
@@ -95,6 +109,7 @@ function AccessIcon(props: { restrictionUiType: RestrictionUiType }) {
  */
 export function useGetRestrictionUiType(
   entityId: string,
+  options: { enabled: boolean },
```

Comment:
Make the parameter more specific, since it doesn't apply to the other query hooks called here. Also, ideally use the full type if you can. It would probably look something like this:
```suggestion
  useGetIsExternalFileHandleOptions: Partial<UseQueryOptions<EntityBundle, SynapseClientError, boolean>>
```